src/v/cluster/topics_frontend.cc

```diff
@@ -81,6 +82,28 @@ std::vector<std::string_view> get_enterprise_features(
     return features;
 }
 
+std::vector<std::string_view> get_enterprise_features(
+  const cluster::metadata_cache& metadata,
+  const cluster::topic_properties_update& update) {
+    const auto& properties = metadata.get_topic_metadata_ref(update.tp_ns)
+                               ->get()
+                               .get_configuration()
+                               .properties;
```

Comment:
fyi: https://github.com/redpanda-data/redpanda/pull/24083/commits/ca957f825d3af6cced4ad123d2bce68d5ffe1a57

Reply:
I've added your two commits, so that I don't break dev